src/types.ts

```diff
@@ -13,7 +13,7 @@ interface CommandExecuteOptions {
 type CommandExecute = (options: CommandExecuteOptions) => Promise<any> | any;
 
 export type CoCommandOptions = {
-    data: SlashCommandBuilder;
+    data: Partial<SlashCommandBuilder>;
```

Comment:
**question:** What does this change do?

Reply:
I see, looks like `addStringOption` returns a different type - we can keep your change